lib/fauna/page.rb

```diff
@@ -221,6 +221,38 @@ def reverse_each
       end
     end
 
+    ##
+    # Returns the flattened contents of the set as an array.
+    #
+    # Ideal for when you need the full contents of a set with a known small size. If you need to iterate over a set
+    # of large or unknown size, it is recommended to use +each+ instead.
+    #
+    # The set is paged in the +after+ direction.
+    def set_contents
```

Comment:
Name suggestions: 

- `flatten`
- `all`
- `all!`
- `to_a` (ruby standard explicit array conversion).

`all!` is nice because it sounds dangerous... but that seems to abuse the normal semantics around `!`.